src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/ReferenceAssemblies.cs

```diff
@@ -995,6 +995,12 @@ public static class Net
                         ImmutableArray.Create(
                             new PackageIdentity("Microsoft.macOS.Ref", "12.3.2372"))));
 
+            private static readonly Lazy<ReferenceAssemblies> _lazyNet70Android =
+                new Lazy<ReferenceAssemblies>(() =>
+                    Net70.AddPackages(
+                        ImmutableArray.Create(
+                            new PackageIdentity("Microsoft.Android.Ref.33", "33.0.68"))));
```

Comment:
💭 Is there an easy way to know if this is the correct version?

Reply:
@sharwell I just used the latest stable that shows as supporting net7.0 on NuGet.org.
Not sure if you have any concerns, or *why* a version could be wrong?